events/events-team/marketing/README.md

```diff
@@ -32,6 +32,8 @@ As Marketing Lead, you are responsible for the overall marketing communications
 - Determine signage needs and copy, coordinate with CNCF on explicit needs  
 - Create/update deck templates for curated talks and/or other purposes
 
+## Shadow to the Marketing Lead Role
+This role can include a shadow role as well. The Marketing Lead can delegate/assign the tasks to the shadow.The expectation from a shadow is to lead one of the events-team in the next summit. The shadow to the Marketing Lead is responsible for handling all Registration lead related activities in the absence of the Lead. 
```

Comment:
Copy/paste error: should be "...responsible for handling all Marketing lead related activities..."

Reply:
good catch :) fixed it.